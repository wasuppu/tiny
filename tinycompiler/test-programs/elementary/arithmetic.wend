main() {
    println +3 + 5 * -2;
    println 3 - 4 * 5;
    println 3 - 10 / 5;
    println (-2+3*4)+5/(7-6)%8;
    println 5<3;
    println 3==3;
    println 3 * (4 + 5) / 7 == 3;
    println true && false;
    println true && false || true;
    println !true;
    println 3<=3 && 3>=3;
}

