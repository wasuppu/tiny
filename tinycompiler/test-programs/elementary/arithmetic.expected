-7
-17
1
15
false
true
true
false
true
false
true
